conventions/documentation/create.md

```diff
@@ -16,17 +17,62 @@ parent: Conventions
 
 ---
 
-## Creating from Create button
+## Create button types
 
-**Create from button (YAML only):**
+### Single Create Buttons
+
+*Single resource create buttons appear in the upper right of the page.*
+![Single create button](../images/create-button-single.png)
+
+---
+
+### Dropdown Create Buttons
+
+*If multiple resources or multiple methods (or both) can be created from a button, a dropdown is used.*
+
+**Multiple methods to create one object**
+- Name of resource is included on button.
+- Name of resource is not included in methods to create it.
+- Methods are recommended to start with “With” or “From”.
+- “From YAML” should appear last, when applicable.
+
+![Dropdown create button multimethod 2](../images/create-button-dropdown-multiMethod2.png)
```

Comment:
small nit: This changed to 
`With form`
`With data upload form`

Reply:
![Create-dropdown](https://user-images.githubusercontent.com/44577420/89374654-b4171f80-d6f4-11ea-85b5-b5aa992731b1.png)

Reply:
Sounds good, I updated that.